### ISSUE TITLE
Shows tags with zero matches.

### ISSUE THREAD

**Issue by taitems** [luis-almeida/filtrify#6]
This is more of a UX choice than anything. Personally I would want to only be able to see the tags that _do_ have matches. Whether you still deliver both options (do/don't show empty matches), I at least think the default option should be that they are hidden. Thoughts?

**Comment by taitems** [comment]
Sorry, just spotted this is a duplicate of issue #3. There were no issues logged when I wanted to add these yesterday.

You said you weren't sure as when there's zero matches it would appear empty? How about showing a "no results" message?

**Comment by luis-almeida** [comment]
Just like you said this was a UX choice.
I realize now that I should at least include it as an option or probably make it like that by default.
And ya, showing "no results" was exactly what I had in mind :)
But I still think I should also disable the search box, making it clear that you can't search anything in there when you have only 0 matches.

**Comment by taitems** [comment]
Wouldn't disabling or hiding the search box mean that the user cannot "x" out (delete) their search query that got found the zero results?

**Comment by luis-almeida** [comment]
No, I mean disabling the search box when you add a tag and after that all the other tags display 0.
A good example is in the "Multiple categories" demo, when you select "Christopher Nolan" in the director panel all the other tags display 0. If we're going to hide all those tags I think I should also disable the search box because there's nothing to searched there.
To "enable" the search box again you just have to remove or "delete" the tag you just added.
